### ISSUE TITLE
Documentation on TomoMaster features

### ISSUE THREAD

**Issue by etienne-napoleone** [tomochain/docs#76]
**Estimated time:** 4h

---------------

**What:**
This will be in the same line as the test you were given to become technical writers.
We need to have some explaining on how to use and contribute to TomoMaster.
This part will be about the TomoMaster features and how to use them.
It should include:
- [Home](https://master.testnet.tomochain.com/)
  -  Network Status
  -  Candidates list
  - Voting
  - Unvoting
- [Candidate](https://master.testnet.tomochain.com/candidate/0xfc5571921c6d3672e13b58ea23dea534f2b35fa0)
  - General information
  - Metrics
  - Rewards
  - Voters
  - Signs
  - Transactions
- [Voter](https://master.testnet.tomochain.com/voter/0x487d62d33467c4842c5e54eb370837e4e88bba0f)
  - General information
  - Candidates
  - Rewards
  - Transactions
- [Settings](https://master.testnet.tomochain.com/setting)
  - Network provider
  - Account infos
- [Apply](https://master.testnet.tomochain.com/apply)
- Search

Note: don't hesitate to use https://squidfunk.github.io/mkdocs-material/extensions/admonition/ to highlight some elements. The plugin is already present, you can just use the syntax.

**Example:**
```markdown
# Search
On the top of the page, you can find a search bar.
You can use this bar to search for a Candidate or Voter by it's address.

!!! tip
    The search will not work if your input is not an address
```

**Where:**
It will be on https://docs.tomochain.com/ in the category "TomoMaster" (not yet present) and named "Features"

**Comment by bluebottleco** [comment]
The file is called tomomaster.md on branch tw/76

https://github.com/tomochain/docs/tree/tw/76/docs/general

**Comment by etienne-napoleone** [comment]
merged in #81
